src/commands/tree_of.rs

```diff
@@ -54,15 +80,45 @@ pub async fn tree_of(matches: &ArgMatches, repo: Repository, config: &Configurat
         env: &additional_env,
     };
 
-    repo.packages()
+    let dot = matches.get_flag("dot");
+
+    let package_dags = repo
+        .packages()
         .filter(|p| pname.as_ref().map(|n| p.name() == n).unwrap_or(true))
         .filter(|p| {
             pvers
                 .as_ref()
                 .map(|v| v.matches(p.version()))
                 .unwrap_or(true)
         })
-        .map(|package| Dag::for_root_package(package.clone(), &repo, None, &condition_data))
+        .map(|package| Dag::for_root_package(package.clone(), &repo, None, &condition_data));
+
+    if dot {
+        for dag in package_dags {
+            let petgraph = convert_dag_to_petgraph(dag.unwrap());
+
+            fn get_edge_color(weight: &DependencyType) -> &str {
+                match weight {
+                    DependencyType::Build => "orange",
+                    DependencyType::Runtime => "blue",
+                }
+            }
+
+            let dot = Dot::with_attr_getters(
+                &petgraph,
+                &[
+                    petgraph::dot::Config::EdgeNoLabel,
+                    petgraph::dot::Config::NodeNoLabel,
+                ],
+                &|_, nr| format!("color = {} ", get_edge_color(nr.weight())).to_string(),
```

Comment:
We could perhaps consider using dashed lines for build dependencies (as they're only required temporarily during compile time) to make it a bit more intuitive?
What do you think @christophprokop?